README.md

```diff
@@ -56,7 +56,7 @@ See also [Getting Started](http://kemalcr.com/guide/).
 - Middleware support
 - Built-in JSON support
 - Built-in static file serving
-- Built-in view templating via [Kilt](https://github.com/jeromegn/kilt)
++ Built-in view templating via [ECR](https://crystal-lang.org/api/1.0.0/ECR.html)
```

Comment:
The URL should be version-agnostic: https://crystal-lang.org/api/ECR.html

Reply:
Should I undo the changes in the README now that Kilt is still being used (deprecated)?